web/over_react_redux/examples/dev_tools/store.dart

```diff
@@ -44,17 +44,7 @@ class BigDecrementAction extends Action {
 ///
 /// Note the use of the special class [DevToolsStore] and the middleware passed
 /// into the store.
-Store store = DevToolsStore<CounterState>(stateReducer, initialState: CounterState.defaultState(), middleware: [remoteDevtools]);
-
-/// DevTools object configured with the server port.
-var remoteDevtools = RemoteDevToolsMiddleware('127.0.0.1:8000');
-
-/// Method used to assign the store and connect the DevTools.
-Future<void> initDevtools() async {
-  remoteDevtools.store = store;
-  window.console.log('Navigate to 127.0.0.1:8000 in order to see Redux DevTools.');
-  await remoteDevtools.connect();
-}
+Store store = DevToolsStore<CounterState>(stateReducer, initialState: CounterState.defaultState(), middleware: [overReactReduxDevToolsMiddleware]);
```

Comment:
#nit breaking this up into multiple lines would aid in readability:
```suggestion
Store store = DevToolsStore<CounterState>(
  stateReducer,
  initialState: CounterState.defaultState(),
  middleware: [overReactReduxDevToolsMiddleware],
);
```